CMakeLists.txt

```diff
@@ -94,29 +91,31 @@ else()
   set(DYN32 OFF CACHE BOOL "Disable support for 32bit fast physics in CCPP")
 endif()
 
-list(APPEND _fv3dycore_defs_private CCPP)
-
-if(OpenMP_Fortran_FOUND)
+if(OPENMP)
   list(APPEND _fv3dycore_defs_private OPENMP)
 endif()
 
 set_property(SOURCE atmos_cubed_sphere/model/nh_utils.F90 APPEND_STRING PROPERTY COMPILE_FLAGS "${FAST}")
 set_property(SOURCE atmos_cubed_sphere/model/fv_mapz.F90  APPEND_STRING PROPERTY COMPILE_FLAGS "${FAST}")
 
 set_target_properties(fv3dycore PROPERTIES Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/mod)
+target_include_directories(fv3dycore INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/mod>
+                                               $<INSTALL_INTERFACE:mod>)
 
 target_compile_definitions(fv3dycore PRIVATE "${_fv3dycore_defs_private}")
+
+#  So much for being consistent:
+# atmos_cubed_sphere/tools/fv_diagnostics.F90: #include <fv_diagnostics.h>
+# atmos_cubed_sphere/tools/fv_eta.F90:         #include <tools/fv_eta.h>
```

Comment:
What do these comments mean?

Reply:
Remainder, explanation why `fv3dycore` target needs both `atmos_cubed_sphere` and `atmos_cubed_sphere/tools` must be specified as include directories. 

`tools/fv_eta.F90` needs `-Iatmos_cubed_sphere` because it includes `<tools/fv_eta.h>`
while
`tools/fv_diagnostics.F90` needs `-Iatmos_cubed_sphere/tools` because it includes `<fv_diagnostics.h>`

Not very consistent.